app/scripts/controllers/home.js

```diff
@@ -8,30 +8,82 @@
  * Controller of the noteeApp
  */
 angular.module('noteeApp')
-  .controller('HomeCtrl', function ($scope) {
-    console.log("Home Page");
+    .controller('HomeCtrl', function ($scope, $uibModal, $log) {
+  $scope.texts = [];
+  $scope.cards = [];
 
-    $scope.stubbedData = {
-    	cards: [],
-    	config: {}
+  $scope.open = function () {
+    var modalInstance = $uibModal.open({
+      templateUrl: 'myPlainText.html',
+      controller: 'ModalInstanceCtrl1'
+    });
+
+    modalInstance.result.then(function (text) {
+      $scope.texts.push(text);
+    }, function () {
+      $log.info('Modal dismissed at: ' + new Date());
+    });
+  };
+
+  $scope.open2 = function () {
+    var modalInstance = $uibModal.open({
+      templateUrl: 'mylist.html',
+      controller: 'ModalInstanceCtrl2'
+    });
+
+    modalInstance.result.then(function (card) {
+      $scope.cards.push(card);
+    }, function () {
+      $log.info('Modal dismissed at: ' + new Date());
+    });
+  };
+
+   $scope.stubbedData = {
+     cards: [],
+     config: {}
     };
     var card = {
-    	title: "card title ",
-    	desc: "card description ",
-    	imageUrl: "https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcTjczsATtMa4mkLl1OvY-6BJit4eOX5dDqvh0Wb56P1xaQVYdd-"
+     title: "card title ",
+     desc: "card description ",
+     imageUrl: "https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcTjczsATtMa4mkLl1OvY-6BJit4eOX5dDqvh0Wb56P1xaQVYdd-"
     }
 
     for(var i=0; i<30; i++){
-    	//var newCard = card;
```

Comment:
this code is from ananth.  He uses it to display the example cards. 

Sent from my iPhone

> On Nov 15, 2015, at 11:25 PM, Elcin <notifications@github.com> wrote:
> 
> In app/scripts/controllers/home.js:
> 
> >      }
> >  
> >      for(var i=0; i<30; i++){
> > -    	//var newCard = card;
> Hey Sharilyn, thanks for the PR. It all looks good. 
> I'm just curious why did you create 30 cards here?
> 
> —
> Reply to this email directly or view it on GitHub.
>